### ISSUE TITLE
Most Recent Commits to RDMTY broke the build.

### ISSUE THREAD

**Issue by christianallred** [billyvg/OverlayPlugin-themes#14]
I am working on a fix right now. 

On a side note here. it looks like the most recent commits that were made switch half of the header data to be personal data and the other half is still combatant. This is probably because the encounter ojbect data does not contain all of the relevant information. IE DirectHit is not included in the info. It looks like rainbow mages overlay plugin will need some updates?

There are a couple of other items that are apparetny being calculated wrong. 

In the encounter object. I am getting 1054 swings 100 crithits and 0% crithit%

I will report my findings on the rainbow mage page as well. but in the meantime. I have moved the encounter information header back to being encounter data and not personal data. I think if we want to be adding in the direct hit information we shoudl wait on the overlay plugin to get updated to inlclude that in the encounter list. instead of just using "self" information and mixing our data sets. 

We could total them on the fly? but that might affect performance. I will take a look into that.

**Comment by billyvg** [comment]
Ah you're right, I hastily merged the PR. I have not been active in FFXIV so it's hard for me to keep this maintained.

**Comment by christianallred** [comment]
I submitted a new pr but it will need updating again once the act plugin is updated

**Comment by christianallred** [comment]
If you need help maintaining this repo I am down.

**Comment by christianallred** [comment]
@billyvg  was rdmty your overlay?

**Comment by deathtator** [comment]
@christianallred  Here is a small list of minor problems I've found

\overlay\css\torgan .css: Missing handles for rdm and sam

\overlay\kyitrai.html: Missing handles for rdm and sam + the image source location.

\overlay\elizar2006_miniparse.html: The source image file for the job icons are too large.

\overlay\Hibiyasleep_Kagerou_pointer.html I believe there is a missing piece of software causing this to not work.

\overlay\css\irealiTY-overlayskin: Missing handles for Rdm and Sam (color value is in hex just like rdmty)'

\overlay\miniparse####.html: Every variant is  missing Handles for Rdm and Sam

**Comment by christianallred** [comment]
@deathtator Hey thanks for the update. When i can i will take a look at each of these. we (@jquick and myself) have updated the rdmty along with  to get that on ready for 4.0 (one more PR outstanding), but haven't seen any other PR's from the owners of the other overlays. I am gonna open an issue and tag them all in them and see who is still planning on maintaining what going forward.

**Comment by billyvg** [comment]
Yeah rdmty was mine
